src/components/CustomStatusBar/types.ts

```diff
@@ -0,0 +1,3 @@
+type CustomStatusBar = () => React.ReactNode;
```

Comment:
You need to add `displayName` property to this type as well, similar to this [PR](https://github.com/Expensify/App/pull/29744/files). [Reasoning](https://github.com/Expensify/App/pull/29744#issuecomment-1810750551). We discussed it some time ago that when there are no props and it is a file with platform specific implementations we need to type the whole component